x/concentrated-liquidity/types/keys_test.go

```diff
@@ -0,0 +1,36 @@
+package types_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/osmosis-labs/osmosis/v15/x/concentrated-liquidity/types"
+)
+
+// TestReverseRelationTickIndexToBytes tests if TickIndexToBytes and TickIndexFromBytes
+// succesfully converts back to the original value.
+func TestReverseRelationTickIndexToBytes(t *testing.T) {
+	tests := map[string]struct {
+		tickIndex int64
+	}{
+		"positive tick index": {
```

Comment:
Would be great if we can add the following cases:
- 0 tick
- Minimum tick
- Maximum tick

Reply:
Thanks for the suggestions! Added according test cases in https://github.com/osmosis-labs/osmosis/pull/5266/commits/51fce1f3393cc2e43e9de0e707d86ad5ca561b34